### DISCUSSION TITLE
Solved: Conditional that says "if session is active"

### DISCUSSION THREAD

**Discussion by shmup** [scandum/tintin#107] [Script help]
I'm keeping everything in a single `.tin` script for simplicity, and I want to reload the script while a session is running, but have some of it scoped to an `if` conditional. I'm just not sure what this conditional would look like.

`#if {"session is active"} {"do some stuff"};#else {"do some other stuff"}`

Generally this is so I don't rerun `#session` bit, but also other ideas in mind.

Thanks!

**Comment by scandum** [comment]
If the name of your session is 'bla' you could use:
```
#if {"@bla{}" === ""} {#showme session exists};#else {#showme session does not exist}
```
